### ISSUE TITLE
There will be an error when obtaining Pinia in the plugin. It used to be fine, but suddenly it doesn't work today.

### ISSUE THREAD

**Issue by bbhxwl** [nuxt/nuxt#29748]
### Environment

- Operating System: Windows_NT
- Node Version:     v20.17.0
- Nuxt Version:     3.13.2
- CLI Version:      3.13.2
- Nitro Version:    2.9.7
- Package Manager:  yarn@1.22.19
- Builder:          -   
- User Config:      devt
ools, ssr, devServer, im
ports, css, extends, mod
ules, runtimeConfig, com
patibilityDate, hooks, components
- Runtime Modules:  @nux
t/ui@2.18.7, @pinia/nuxt
@0.6.1, @pinia-plugin-pe
rsistedstate/nuxt@1.2.1,
 dayjs-nuxt@2.1.11, @nux
tjs/device@3.2.4, @nuxtj
s/color-mode@3.5.2, @nux
tjs/tailwindcss@6.12.1, 
@vant/nuxt@1.0.6, @vueuse/nuxt@11.1.0

### Reproduction

 "getActivePinia()" was called but there was no active Pinia. Are you trying to use a store before calling "app.use(pinia)"? See https://pinia.vuejs.org/core-concepts/outside-component-usage.html for help. This will fail in production.
![Image](https://github.com/user-attachments/assets/fc6b9b82-02c7-4858-a061-cc2c231e2822)

### Describe the bug

 "getActivePinia()" was called but there was no active Pinia. Are you trying to use a store before calling "app.use(pinia)"? See https://pinia.vuejs.org/core-concepts/outside-component-usage.html for help. This will fail in production.
![Image](https://github.com/user-attachments/assets/4aeae0d9-0935-4ab5-86dc-31356bea29bf)

**Comment by danielroe** [comment]
I do not believe this was a Nuxt issue. if it's still occurring, please report upstream to pinia.
